src/drive/web/modules/navigation/breadcrumb.styl

```diff
@@ -32,15 +31,15 @@
     height     $coz-bar-size
     border     0
     flex-shrink 0
-    background embedurl('../assets/icons/icon-arrow-left.svg') center center no-repeat
+    background embedurl('../../../assets/icons/icon-arrow-left.svg') center center no-repeat
```

Comment:
Why this change.

Reply:
This is so weird. Once I removed `@require '../../../styles/topbar.styl'`, the build started failing… because indeed theses pathes were wrong. Why do they work when the `@require '../../../styles/topbar.styl'` line is present is beyond me.